src/i18n/en/translation.json

```diff
@@ -289,6 +289,7 @@
         "subjectOfResponses": "Subject of Responses"
       },
       "invalidRespondent": "You are not authorized to complete this assessment on behalf of this subject.",
+      "mismatchedRespondent": "There's a mismatch between the expected responder and the signed-in web app user. Log out of the web app and login with the responder’s login to initiate Take Now again.",
```

Comment:
> and login with the responder’s login to initiate Take Now again.

My only issue with this copy is that it seems to imply that that logging out and logging into the responder's login re-initiates Take Now. But it doesn't cause Take Now to be re-initiated after you log in. You still have to manually close the tab and redo Take Now as an additional manual step. So I would probably reword this as:

`There's a mismatch between the expected responder and the signed-in web app user. Log out of the web app and log in with the responder’s login, then re-initiate Take Now.`

Reply:
@farmerpaul I agree with you. But this copy came from the ticket. Can you leave that feedback on the ticket please and tag Cari?

Reply:
@farmerpaul I applied the copy change. Thanks for the suggestion 👌 I decided to reword the last bit from _"admin panel"_ to _"Admin App"_, since that's the wording used on the pop-up at the end of the activity